### PR TITLE
Build kube-proxy image

### PR DESCRIPTION
**Pull request by pecameron**
This image is hyperkube with conntrack-tools and iptables

SDN-310 Create origin kube-proxy image
https://jira.coreos.com/browse/SDN-310

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@smarterclayton @sdodson @squeed PTAL

**Comment by smarterclayton** [comment]
/hold

Need the ART repo created, and then link to ocp-build-data.  Also, you need to be built on top of my PR to ocp-build-data that reorders parents

**Comment by smarterclayton** [comment]
Another related change #22082 but isn't in 4.0

**Comment by pecameron** [comment]
@smarterclayton Could you tell me more about "built on top of my PR to ocp-build-data that reorders parents"? How do I do this?

File: images/kube-proxy/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+reviewers:
```
Comment (dcbw) [inline images/kube-proxy/OWNERS]: @pecameron could you add people from the net team to the image OWNERS too? As long as @smarterclayton thinks that's OK of course. To both Reviewers and Approvers.

Comment (pecameron) [reply images/kube-proxy/OWNERS]: Done @dcbw @squeed PTAL

**Comment by pecameron** [comment]
@smarterclayton ci/prow/e2e-aws failed with
evel=fatal msg="waiting for Kubernetes API: context deadline exceeded"

**Comment by pecameron** [comment]
ci/prow/e2e-aws — Job succeeded.
next test run...
/test e2e-aws

**Comment by pecameron** [comment]
ci/prow/e2e-aws — Job succeeded.
rerun test
/test e2e-aws

**Comment by pecameron** [comment]
Failed before network started
clusteroperator/kube-controller-manager changed Failing to True: StaticPodsFailing: StaticPodsFailing: nodes/ip-10-0-158-188.ec2.internal

/test e2e-aws

**Comment by pecameron** [comment]
Failed: StaticPodsFailing: nodes/ip-10-0-158-188.ec2.internal
/test e2e-aws

**Comment by pecameron** [comment]
Succeeded: 3
ci/prow/e2e-aws — Job succeeded.
/test e2e-aws

**Comment by pecameron** [comment]
Succeeded: 4
ci/prow/e2e-aws — Job succeeded.
/test e2e-aws

**Comment by pecameron** [comment]
Succeeded: 5
ci/prow/e2e-aws — Job succeeded.
/test e2e-aws

**Comment by pecameron** [comment]
Succeeded: 6
ci/prow/e2e-aws — Job succeeded.
/test e2e-aws

**Comment by pecameron** [comment]
Failed: kube-apiserver Kube API started failing
Succeeded: 6

/test e2e-aws

**Comment by pecameron** [comment]
@smarterclayton @dcbw what more needs to be done before this merges?
This is just hyperkube with 2 additional rpms that are not used in the image.

**Comment by pecameron** [comment]
@sdodson PTAL

**Comment by pecameron** [comment]
@smarterclayton @sdodson Can this be merged? Is there more to do?

**Comment by pecameron** [comment]
@smarterclayton Can this be merged now? It blocks ART-487.

**Comment by pecameron** [comment]
@smarterclayton Can this be merged now? It blocks ART-487.

**Comment by sdodson** [comment]
/hold cancel
The referenced ART-487 seems to address Clayton's concerns, please make sure that we follow through with everything required therein.

**Comment by pecameron** [comment]
@sdodson I'm confused (again). I thought this PR had to go in to unblock ART-487. I thought @smarterclayton was holding off on the merge to let some other items happen first. As far as I know the kube-proxy image is supposed to be part of 4.1 (needed by Callico) and we are hard upon the date. 
@knobunc this is the pr we discussed at lunch.

**Pull request merged**